me.mitya57.ReText.json

```diff
@@ -0,0 +1,378 @@
+{
+    "app-id": "me.mitya57.ReText",
+    "base": "io.qt.qtwebkit.BaseApp",
+	"base-version": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "retext",
+    "rename-icon": "retext",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=host",
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--env=QTWEBENGINE_DISABLE_SANDBOX=1",
+        "--env=QT_XCB_FORCE_SOFTWARE_OPENGL=1",
```

Comment:
I've not seen other apps require that and sounds like it will kill performance.

Reply:
Qt Web Engine: stability problems may occur when using nouveau or intel driver
https://github.com/lirios/browser/issues/77
https://lists.freedesktop.org/archives/flatpak/2017-June/000722.html